File: prime_print.py
def ask():
	start = int(raw_input("Give me a starting number"))
	stop = int(raw_input("Give me an ending number"))

	return[start,stop]

#YOU CAN NEVER HAVE A FUNCTION CALLED AS ARGUMENT-  ASK IS A NUMBER VALUE BELOW!
def prime_print():
	array = ask()

	prime_array = []

	i = array[0]
	j = array[1]

	while i < (j+1):
		
		if (i <=3 and i <=1):
			prime_array.append(i)
		elif i%2 != 0:
			k = 3
			flag = False
			while k < i:
				if i/k !=0:
					flag = True
				else:
					flag = False
					break
				k +=1
			if flag == True:
				prime_array.append(i)
		i +=1

	return prime_array

print(prime_print())


				

File: prime_print.rb
def ask()
	puts("Give me a start number")
	start = gets.chomp.to_i

	puts("Give me a stop number")
	stop = gets.chomp.to_i

	return[start, stop]
end

def prime_print()
	array = ask()

	j = array[0]
	
	prime_array = []
	while j < (array[1]+ 1)
		flag = false
		if  (j <=3 && j >=1)
			flag = true
			prime_array.push(j)
		elsif j%2 != 0
			i = 3
			while i < j
				if j%2 != 0
					if j/i != 0
						flag = true
					else 
						flag = false
					end
				end
				i +=1
			end
			if flag == true
					prime_array.push(j)
			end
		end
		
		j +=1
	end
	
	return prime_array	

end

puts(prime_print)
